src/app/config-page/config-page.component.html

```diff
@@ -17,86 +17,93 @@
-              <span class="formLabel">Speak button always shown</span>
-            </mat-checkbox>
-          </mat-card>
-          <mat-card>
-            <mat-checkbox [(ngModel)]="displayedButtons.showBackspaceButton">
-              <span class="formLabel">Backspace button always shown</span>
-            </mat-checkbox>
-          </mat-card>
-          <mat-card>
-            <mat-checkbox [(ngModel)]="displayedButtons.showClearButton">
-              <span class="formLabel">Clear button always shown</span>
-            </mat-checkbox>
-          </mat-card>
-          <mat-card>
-            <mat-checkbox [(ngModel)]="displayedButtons.showHomeButton">
-              <span class="formLabel">Home button always shown</span>
-            </mat-checkbox>
-          </mat-card>
-        </div>
-      </mat-tab>
-      <mat-tab label="Scanning">
-        <div class="formContainer">
-          <mat-card>
-            <mat-checkbox [(ngModel)]="scanningConfig.enabled">
-              <span class="formLabel">Enable scanning</span>
-            </mat-checkbox>
-          </mat-card>
-          <mat-card>
-            <mat-form-field class="formInputField">
-              <input [(ngModel)]="scanningConfig.time" matInput placeholder="Time to stay on each scanned item (in milliseconds)"/>
-            </mat-form-field>
-          </mat-card>
-        </div>
-      </mat-tab>
-      <mat-tab label="Apperance">
-        <div class="formContainer">
-          <mat-card>
-            <div>
-              <mat-slider min="1" max="10" step="1" color="Primary"
-                [(ngModel)]="appearanceConfig.borderThickness" thumbLabel="true"
-                (input)="borderThicknessChange($event)"
-              ></mat-slider>
-              <div class="borderExample" [style.border-width]="appearanceConfig.borderThickness + 'px'">Border Thickness</div>
-            </div>
-          </mat-card>
-        </div>
-      </mat-tab>
-    </mat-tab-group>
+    <form (ngSubmit)="save()" #configForm="ngForm">
+
+      <div class="formButtons">
+        <button type="submit" mat-raised-button color="primary" [disabled]="!configForm.form.valid">Save</button>
+      </div>
+      <mat-tab-group>
+        <mat-tab ngModelGroup="boardURL" #boardURLCtrl="ngModelGroup">
+          <ng-template mat-tab-label>
+            <mat-icon *ngIf="boardURLCtrl.invalid" class="invalidTab">error</mat-icon>
+            <span [class.invalidTab]="boardURLCtrl.invalid">Board URL</span>
+          </ng-template>
+          <div class="formContainer">
+            <mat-card>
+              <mat-form-field class="formInputField">
+                <input [(ngModel)]="boardURL" required matInput name="boardURL" placeholder="URL for obf/obz file" class="formInputUrl">
+              </mat-form-field>
+              <div class="clickable" (click)="copyToClipboard()"><i class="material-icons">link</i>&nbsp;Create link</div>
```

Comment:
We should probably be consistent about how we use icons

Reply:
Unfortunately replacing this one seems to mess up the alignment and is not trivialy fixable...